include/rsl/static_string.hpp

```diff
@@ -0,0 +1,97 @@
+#pragma once
+
+#include <rclcpp/parameter_value.hpp>
+
+#include <array>
+#include <cassert>
+#include <string>
+#include <string_view>
+#include <type_traits>
+
+namespace rsl {
+
+/**
+ * @brief Fixed size string with conversion operators for std::string_view, std::string, and
```

Comment:
```suggestion
 * @brief Fixed capacity string with conversion operators for std::string_view, std::string, and
```
Can we be more clear about where we mean size and where we mean capacity such that we match how the standard uses these terms?